src/async_block_store.rs

```diff
@@ -23,17 +25,20 @@ impl AsyncTempAlias {
 #[derive(Debug, Clone)]
 pub struct AsyncTempAlias(Arc<TempAlias>);
 
-pub trait Unblocker {
+/// Adapter for a runtime such as tokio or async_std
+pub trait Runtime {
     fn unblock<F, T>(&self, f: F) -> BoxFuture<T>
```

Comment:
given the name change of the trait, why not `run`?

Reply:
The name was inspired by https://crates.io/crates/blocking

Reply:
run does not properly communicate that the block to be run will most likely contain a long, blocking IO operation.

Reply:
Makes sense